index.js

```diff
@@ -50,15 +50,16 @@ module.exports = function (sails) {
 
         initConnections: function () {
             var connections = {}, connection, connectionName;
+            var datastore = sails.config.models.datastore || 'default';
 
-            sails.log.verbose('Using default connection named ' + sails.config.models.connection);
-            if (!sails.config.connections[sails.config.models.connection]) {
-                throw new Error('Default connection \'' + sails.config.models.connection + '\' not found in config/connections');
+            sails.log.verbose('Using default connection named ' + datastore);
+            if (!sails.config.datastores[datastore]) {
+                throw new Error('Default connection \'' + datastore + '\' not found in config/connections');
             }
 
-            for (connectionName in sails.config.connections) {
+            for (connectionName in sails.config.datastores) {
 
```

Comment:
And then use it everywhere like here:
```
for (connectionName in sails.config[connectionKey + 's']) {
...
```
This allow us for supporting both pre v1.0 versions along with v1.0